docs/sphinx/reference-libobs-graphics-graphics.rst

```diff
@@ -989,6 +989,8 @@ Texture Functions
    :param strides:      Array of size *n_planes* with the stride of each plane
    :param offsets:      Array of size *n_planes* with the offset of each plane
    :param modifiers:    Array of size *n_planes* with the modifier of each plane
+                        This array must contain the modifier of the dmabuf for each plane.
+                        Passing different modifiers is not supported.
```

Comment:
Please move this above, as a new paragraph right after the `can fail to be created or rendered.` line.

Allow me to suggest the following:

> All modifiers passed in the *modifiers* array must be equal. Passing different modifiers for each plane is unsupported.